packages/modals/src/styled/StyledHeader.ts

```diff
@@ -28,6 +31,11 @@ export const StyledHeader = styled.div.attrs<IStyledHeaderProps>({
   margin: 0;
   border-bottom: ${props => props.theme.borders.sm} ${getColor('neutralHue', 200)};
   padding: ${props => `${props.theme.space.base * 5}px ${props.theme.space.base * 10}px`};
+  ${props =>
+    props.isCloseButtonPresent &&
+    `padding-${props.theme.rtl ? 'left' : 'right'}: ${
+      props.theme.space.base * (baseMultipliers.size + baseMultipliers.side)
```

Comment:
The padding added only borders the `Close` button, no additional multiplier is added. We may want to add additional padding (eg `Sheet` has added `2` to the base multiplier for additional breathing room with the button)

Reply:
Of course, that makes sense. Will update.

Reply:
Would we want to treat the `left`/`right` values the same here? It seems to differ from `Sheet` (`6.5` vs `2`)

Reply:
No, it should be different, due to the fact that padding is larger in `Modal` than it is in both `DrawerModal` and `Sheet`. The end result is that the close button should align with buttons in the footer for any of these components.